pkg/controller/containerjfr/resource_definitions/resource_definitions.go

```diff
@@ -247,14 +258,18 @@ func NewExporterService(cr *rhjmcv1alpha1.ContainerJFR) *corev1.Service {
 			Name:      cr.Name,
 			Namespace: cr.Namespace,
 			Labels: map[string]string{
-				"app":       cr.Name,
-				"component": "container-jfr",
+				"app.kubernetes.io/name":       "exporter",
```

Comment:
For resources with their own name, it seems convention to prefix the label `app.kubernetes.io/name` with the crd name, I guess: `cr.Name + "-exporter"`

Reply:
Would you also want this applied to the `container-jfr` Pod resource?

Reply:
They could be set to just `cr.Name`. Or suffixed with deployment/pod, I'm not sure. I'd like to find some documentation on the labels; atm I'm just blindly going off of what the RH Jaeger Operator has done. They seem to have set the Deployment to cr.Name, the ReplicaSet to cr.Name + pod hash and pod to cr.Name + pod hash + another hash that I don't know the origin of.

Reply:
Okay, sounds good. I'll look for some docs or best-practices as well and see if anything turns up. For now I'll leave this sitting WIP as-is.